x-pack/test/functional/apps/reporting_management/report_listing.ts

```diff
@@ -114,7 +116,7 @@ test_user\nCompleted at 2020-04-21 @ 06:55 PM - Max size reached\nreport2csv\n20
 
       // scan page 3
       tableText = await getTableTextFromElement(await testSubjects.find('reportJobListing'));
-      const PAGE_CONTENT_3 = `report1csv\n2020-04-21 @ 06:54 PM\ntest_user\nCompleted at 2020-04-21 @ 06:54 PM - Max size reached`;
+      const PAGE_CONTENT_3 = `report1csv\nsearch\n2020-04-21 @ 06:54 PM\ntest_user\nCompleted at 2020-04-21 @ 06:54 PM - Max size reached`;
```

Comment:
Looks like a regression is here where `search` is not showing up in the table, but it's actually that the es archives had older format of data